spec/lib/gds_api/whitehall_export_spec.rb

```diff
@@ -1,14 +1,21 @@
 # frozen_string_literal: true
 
 RSpec.describe GdsApi::WhitehallExport do
-  describe "document_list" do
-    let(:whitehall_adapter) { GdsApi::WhitehallExport.new(Plek.find("whitehall-admin")) }
+  let(:whitehall_adapter) { GdsApi::WhitehallExport.new(Plek.find("whitehall-admin")) }
+  let(:whitehall_host) { Plek.new.external_url_for("whitehall-admin") }
+  let(:document_id) { "123" }
 
+  describe "document_list" do
```

Comment:
Sorry I know you didn't introduce this and you're probably already aware of the convention. However would be good to prefix this method name with a # as per http://www.betterspecs.org/#describe

Reply:
Done 👍